pine/interfaces

```diff
@@ -0,0 +1,7 @@
+# ifupdown has been replaced by netplan(5) on this system.  See
+# /etc/netplan for current configuration.
```

Comment:
I think we should archive pine directory by moving it to the classroom repo just in case we need it later. It can be removed from here though, pi is documented in the wiki.